checks/microbenchmarks/gpu/amd_gpu/amd_gpu.py

```diff
@@ -32,6 +32,7 @@ def prepare_build(self):
         self.build_system.builddir = f'build_{self.benchmark}'
         self.prebuild_cmds = [f'ln -fs {self.benchmark}/* .', 'pwd']
         self.build_system.max_concurrency = 8
+        self.build_system.generator = 'Ninja'
```

Comment:
This is not valid with `ReFrame` if I am not wrong. We have planned to refactor CMake for a long time though https://github.com/reframe-hpc/reframe/issues/3378